deployment/docker-compose.yml

```diff
@@ -27,14 +27,11 @@ services:
   postgres:
     image: postgres
     container_name: postgres
+    restart: always
```

Comment:
Why?

Reply:
In docker doc we have following notice: `Docker recommends that you use restart policies, and avoid using process managers to start containers.`
The default value is `no`.

```
no                       | Do not automatically restart the container. (the default)

on-failure           | Restart the container if it exits due to an error, which manifests as a non-zero exit code.

always                | Always restart the container if it stops. If it is manually stopped, it is restarted only when Docker daemon restarts or the container itself is manually restarted. (See the second bullet listed in restart policy details)

unless-stopped | Similar to always, except that when the container is stopped (manually or otherwise), it is not restarted even after Docker daemon restarts.
```

Now imagine that you run Edna somewhere on your server and accidentally your server fall down. You restart server and forgot to manually run your containers. With this setting (if your docker daemon starts automatically) postgres container will start automatically. 

I guess we have to add similar option to other services too.

Reply:
Ok, I see that it still doesn't automatically restart if I manually stop the container (e. g. click `Ctrl-C` after `docker-compose up postgres`), so it's fine. This setting doesn't really matter for us because we don't use `docker-compose.yaml` on server, we only use it locally. But perhaps it can be useful for other people who may want to run Edna on their server, so I'm ok with this setting.

Please don't add it _silently_ though, write a sentence or two about why it's done. Ideally it should be put into its own commit because it looks independent of other changes.